src/components/views/rooms/EntityTile.tsx

```diff
@@ -105,10 +104,23 @@ export default class EntityTile extends React.PureComponent<IProps, IState> {
         };
     }
 
+    private getPresenceLabel(): JSX.Element | undefined {
```

Comment:
What is method doing? A bit of documentation is welcomed :)

Reply:
I feel like the method is simple enough that more documentation would be a little bit redundant? Perhaps we could even inline this method.

Reply:
I mean, with some jsdoc, a contributor is able to understand what the method is doing or what is it's purpose without looking at the inner code

Reply:
You're right in general but for this particular method, the method signature (name and return type) gives a good enough idea of what it does. Anyhow I've added some jsdoc :+1: